### DISCUSSION TITLE
How to report outer errors as well?

### DISCUSSION THREAD

**Discussion by HugeLetters** [open-circle/valibot#667] [Q&A] [answered]
I have this schema
```ts
const EncryptedDataSchema =  v.strictTuple(
    [HexStringBufferSchema, HexStringBufferSchema, HexStringBufferSchema],
    'Encrypted data must consist of only 3 hex string segments separated by a colon(:)',
 );
```

If a 2nd element in the tuple will be invalid - I'll get the error about it.
Is there a way to provide the outer error message as well? 

I saw you can set a message with config but it changes the message alltogether.
What I would like to be able to do is to report both the inner error(2nd element is invalid) and the outer error(about encrypted data)

**Comment by fabian-hiller** [comment]
Can you send me a [playground](https://valibot.dev/playground/) link and tell me the current and expected behavior?

**Reply by HugeLetters** [reply]
Here's [playground](https://valibot.dev/playground/?code=JYWwDg9gTgLgBAKjgQwM5wG5wGZQiOAcg2QBtgAjCGQgbgChGBjCAO1XgAkBTADwGUYUYKwDmAIQCu2bNyj8mAC24hkcALyYAdGGBhuACnpxtHYWIMBKADTHtQ5O2zQQBg2csaAfHDNbUYOQwBgBEIZY29JYM9Czs8ACSGAAy3GIwigDCykwA1hraStx5RiYGFNKyUABccACCUFDIAJ4APGYiol6e6j4VMnJapGmiGRrqmgAstiaEAAp4GMAAJtzLcEzATGDKUHAJAGpwimhwIiTk68PpioS20bFsHPWSGQAqyKKpN9nF+ZoYLRFEp2cqVOS1BpNNodMTdbxwfpVIYjMYTKYzIgLCBLVbrZCvRRwGCfY6nc5kFZwa6jW73GJxZ4AUVYTCgzTAMDWABFkCSFMpVAVAbp9KVTEJOlZMYCHE4XG5YaIej4lf5AsBgoRqoQIjL-JKmDA3pJAoYANp2EwivSGHgCSViKQDeRFVTWfYpVFZHK5SIma06W0Ge2CcwScGuwXID11QkfL7e355f0B0OOiMugUqGN2AC6mMILLZHK562WfLUIEkz0ZwGeEGwcDYpGacAAzMc+L4M75uKIQGkYOhUNwwMgmmXEW21CxSGwDNVLLr6cwnvAoNxUJJSPAAf5kLI5hPRwZi+zOTzK9n3SF56RwgxgE2DABCTfb3f+SRMJhb1CeAA3nYjIQMMQwQKIBiAtgpB8lyrAGB+O4wFo9bbluEQMAAvnA3CkKOcDASYoHgfOUEhBAuSPvQ2FAA)
This is the error I would like to get
```ts
{
  root: [ "Encrypted data must consist of only 3 hex string segments separated by a colon(:)" ],
  nested: {
    0: [ "Provided cicpher IV has invalid length" ],
    1: [ "Invalid type: Expected string but received undefined" ],
    2: [ "Invalid type: Expected string but received undefined" ]
  }
}
```

**Reply by fabian-hiller** [reply, marked as answer]
This is only possible with our low level API `rawCheck`. [Here](https://valibot.dev/playground/?code=JYWwDg9gTgLgBAKjgQwM5wG5wGZQiOAcg2QBtgAjCGQgbgCh6BjCAO1XgAkBTADwGUYUYKwDmAIQCu2bNyj8mAC24hkcALyYAdGGBhuACnpxtHYWIMBKADTHtQ5O2zQQBg2csaAfHDNbUYOQwBoSEljb0lgzMbBxwAJIYADLcYjCKAMLKTADWGtpK3LlGJgYU0rJQAFxwAIJQUMgAngA8ZiKiXp7qPuUyclqkqaLpGuqaACy2JoQACngYwAAm3EtwTMBMYMpQCQBqcIpocCIk5GtDaYqEtlEx7PC1kukAKsiiKVdZRXmaGFqFYp2MoVOQ1eqNVrtMRdbxwPqVQbDUbjSbTIjzCCLFZrZDPRRwGDvQ7HU5kZZwS4ja63aIsB5wACirCYUCaYBgqwAIsgiQplKp8v9dPoSqYhB0rOj-g4nC43NDRN0fIr-IFgMFCFUwhETP92kwYC9JIFDABtOx6nR6Qw8AQSsRSfryQqqawJZLIzLZHK6kzaEW2viCcwSUEugXId1PV7vT7U765P0mO0hjpOyr8lRRuwAXT9-0aAHdEzk3ABvOBLXlobgwd3IJZLeKoVCSbhwAC+yrg5ctJ2wcAMAEJq0TUHWtDB2as4AAyOd1BrNLTAVAQ5oGMe1mBaM7t8K9-smRvN1vtgx9-3XuAgbit97cKrHm+EZms9mctbb2+SOL01A13gCBBzYUgmjgABmQ4+F8B1RF8bhRDvVgYHQCcwGQRov3hCC1BYUg2AMKpLBuF9uwYa9OzsbtaUYAD4Cge9JFIeA-n8ZBZFmLCJwMd82Q5bkayzN0iEI0gwgYYBBxHJi21Y-xJCYJh71QTwr3WWIICGQYIFEAx-mwUheU5VgDDkljdzXNt73CBhOzgbhSAnI8TAA7TuF0-TCAgHJJPoaigA) is an example.
